test/runtests.sh

```diff
@@ -19,6 +19,20 @@ if [[ $build_type != "-d" && $build_type != "-t" ]]; then
     elif [[ -f "$test_path/../BuildLinux/Test/ch" ]]; then
         echo "Warning: Test build was found"
         build_type="-t"
+    elif [[ -f "$test_path/../BuildLinux/Release/ch" ]]; then
+        # TEST flags are not enabled for release build
+        # however we would like to test if the compiled binary
+        # works or not
+        CH="$test_path/../BuildLinux/Release/ch"
+        echo "Warning: Release build was found"
+        RES=$(${CH} $test_path/test/basics/hello.js)
```

Comment:
Is hello.js the only file we would be interested in testing? It seems even without test flags we could test a little more than the bare basics to make sure things are all OK.